google-bigquery.yml

```diff
@@ -18,7 +18,8 @@ id: 3d4b5b0c-931d-11ea-a02b-cb6a223f4ab2
 description: Beta - A fast, economical and fully managed data warehouse for large-scale data analytics.
 display_name: Google Big Query (Beta)
 image_url: file://service-images/csb.png
-documentation_url: https://cloud.google.com/bigquery/docs/
+documentation_url: https://docs.vmware.com/en/Cloud-Service-Broker-for-VMware-Tanzu/index.html
```

Comment:
I can see the logic of this, but it does break the open source story. Maybe that's worth doing, but we should discuss it first.

Reply:
Sure! we will talk about it on Monday. Thank you very much @blgm